### PR TITLE
New Result Implementation

### PR DESCRIPTION
**Pull request by N0W0RK**
This is an alternate approach to solve [this issue](https://github.com/N0W0RK/TUM_GAD_Tests_SS21/pull/52#issuecomment-863953872). The Fast result copies the treenodes as a custom List structure.

### CODE REVIEW

File: GAD 8 - Binomial Heap/tests/binomilia/FastResult.java [hunk 1]
```diff
@@ -3,90 +3,100 @@
...
+		return logs.stream().map(e -> e.stream().map(Object::toString).collect(Collectors.joining(", ", "{", "}"))).map(String::valueOf).toList();
+	}
+
+	private List treeToList(BinomialTreeNode tree) {
+
+		List res = new ArrayList<>();
+
+		int rank = tree.rank();
+
+		res.add(tree.getElement());
```
Comment (babyygemperor) [inline GAD 8 - Binomial Heap/tests/binomilia/FastResult.java:93]: There is no method getElement in BinomialTreeNode

**Review by babyygemperor — COMMENTED** [review]
The changes for GAD-7 are working like a charm, really nice changes, love it

**Review by babyygemperor — COMMENTED** [review]
The changes for GAD 6 are also nice, works like a charm

**Comment by N0W0RK** [comment]
This should have actualkly only been a change in result gimme a sec.

**Pull request closed without merging**